chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
So your choice of IDE/editor has to dictate my preferences?  If your IDE has a wider view than my  preference then yours must display the content just fine whereas the inverse will not be true.

@tanabarr: Are you really going to base your entire review of this patch on the subjective whitespace content of a couple of lines, in which the author provides a reasonable explanation of balance between readability and format?

To be fair, the patch where I asked you to fix the continuators had more problems than just whitespace.  I certainly would not reject a patch just for whitespace issues (never mind subjective ones where the author has provided an explanation for the exceptions) and I don't think you will find any occurrence where I have done that.